src/cclib/writer/cjsonwriter.py

```diff
@@ -17,97 +17,424 @@
     has_openbabel = False
 
 import os.path
+import ntpath
 import json
+import numpy as np
 
 from . import filewriter
 
 
 class CJSON(filewriter.Writer):
     """A writer for chemical JSON (CJSON) files."""
+    
+    # The expected Key names for all supported attributes.
+    _attrkeynames = {
```

Comment:
This might be something it would make sense to move to parser/data.py and import here. More a note than something to do now - maybe add a TODO comment, we can do this later.